packages/@webex/plugin-meetings/src/peer-connection-manager/util.js

```diff
@@ -16,4 +42,71 @@ PeerConnectionUtils.convertCLineToIpv4 = (sdp) => {
 
   return replaceSdp;
 };
+
+/**
+ * estimate profile levels for max-fs & max-mbps values
+ * @param {string} sdp
+ * @param {number} maxFsValue
+ * @returns {string}
+ */
+PeerConnectionUtils.estimateProfileLevels = (sdp, maxFsValue) => {
```

Comment:
maybe we should rename the function, because it actually modifies the SDP and that's not reflected in the name, so maybe it should be name something like "adjustH264Profile" or "applyMaxFs" or "applyMaxFsToSdp" or something like that. Basically the function modifies the SDP to make sure that the required maxFs value is used by all h264 codec payload types (and also makes sure maxmbps is good enough for 30 fps), so this should be reflected in the function name.

Reply:
Thanks @marcin-bazyl for comment,
"adjustH264Profile", "applyMaxFsAndMaxMbps" or "applyMaxFsAndMaxMbpsToSdp" sounds good, 
What name you will suggest ?
@arun3528, @marcin-bazyl

Reply:
I think "adjustH264Profile" is my favourite